app/src/main/java/space/taran/arknavigator/ui/extra/ExtraLoader.kt

```diff
@@ -30,4 +33,35 @@ object ExtraLoader {
             else -> {}
         }
     }
+
+    fun loadInfo(
+        meta: ResourceMeta,
+        path: Path,
+        resourceId: TextView,
+        resourceName: TextView,
+        resourcePath: TextView,
+        resourceSize: TextView,
+        resourceResolution: TextView,
+        resourceDuration: TextView
```

Comment:
Can you do it in  more general way?
Something like `kindPlaceholders: List<TextView>`
You can also add check `require(kindPlaceholders.size == 3)`

Reply:
Yes, I will.